doc/user/content/manage/user-management/rbac-intro.md

```diff
@@ -0,0 +1,119 @@
+
+First, we need to make a distinction between authentication and authorization.
+Authentication dictates who can log in to a system while authorization
+determines what a user can access within a system.
+
+Authentication in Materialize is handled when you sign up or are invited to a
+Materialize organization. Your immediate privileges are determined by your
+invitation from an administrator or are assigned automatically by Materialize
+when you sign up.
+
+Authorization is determined in RBAC by your organization administrator. When you
+invite users to your Materialize organization, you have the option to give the
+user more elevated privileges in the Materialize administrator console.
+
+## RBAC Structure
+
+RBAC in practice is a group of roles with assigned privileges and attributes.
+You can assign specific users to roles or assign privileges to users to inherit
+from other roles.
+
+### Roles
+
+A role is a collection of permissions you can apply to users. Roles make it
+easier to assign or revoke privileges on Materialize objects. You can group
+users into specified roles with different levels of permissions and adjust those
+permissions to ensure they have the correct level of access to objects.
+
+### Role Attributes
+
+Role attributes are actions available to any role you create. Attributes are
+independent of any other object in Materialize and apply to the entire
+organization. You can edit these actions when you create the role:
+
+| Name            | Description                                                                     |
+|-----------------|---------------------------------------------------------------------------------|
+| `CREATEDB`      | Can create a database.                                                          |
+| `CREATEROLE`    | Can create, alter, drop, grant membership to, and revoke membership from roles. |
+| `INHERIT`       | Can inherit the privileges of roles that it is a member of. On by default.      |
+| `CREATECLUSTER` | Can create a cluster.                                                           |
+
+### Privileges
+
+Privileges are the actions or operations a role is allowed to perform. When you
+create a role, you can select specific privileges for that role based on the
+objects that role needs to access.
+
+Materialize supports the following privileges:
+
+| Privilege | Description                                                              | Abbreviation  |
+|-----------|--------------------------------------------------------------------------|---------------|
+| `SELECT`  | Allows reading rows from an object.                                      | `r`("read")   |
+| `INSERT`  | Allows inserting into an object.                                         | `a`("append") |
+| `UPDATE`  | Allows updating an object (requires SELECT if a read is necessary).      | `w`("write")  |
+| `DELETE`  | Allows deleting from an object (requires SELECT if a read is necessary). | `d`           |
```

Comment:
We currently have a small limitation where SELECT is always necessary for these two privileges. It might be worth calling that out.